apps/charterafrica/src/components/Comments/YoutubeCommentsJson.js

```diff
@@ -0,0 +1,5130 @@
+// Used for purposes of testing
```

Comment:
What's up with this file?

Reply:
> 👍🏽 ... looks good.
> 
> 1. Most of the comments are just one line, read more doesn't add anything.
> 2. `CommentsList` should just be `Comments`.
> 3. Looks like you're using preprocessed comments structure. Are we able to do this on the fly?

@kilemensi on 3. What do you mean by doing this on the fly? To make the API request?

Reply:
Yeah, I mean we'll be pulling these comments on the fly right? Or what is your plan?

Reply:
Since this is a re usable component outside Youtube comments, I do not expect logic such as API fetch to happen on this component.

Comment fetching can happen at `EbededYoutubeVideo` component higher in the tree.

Reply:
Yes, it will probably happen in api or getServerSideProps but my question was: is the processing we have to do light enough for it to be done on the fly?

---

.. and do we need this file committed with the component?

Reply:
It is an expensive operation because we need to loop through each parent comment to get all the threads.

Also,  since we do not want google API key exposed, then the logic will be best handled on the backend.

Reply:
> each parent comment to get all the threads
>

Wait, there is no single end point that returns all the comments for a given video?

Reply:
Thread replies are capped at 5 per parent comment

Reply:
created a `_mock` directory for such files and added it to `.gitignore`

Reply:
I will delete this before merge